public/site.webmanifest

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "devscard",
```

Comment:
Nie wiem, czy da się to skonfigurować, ale jeśli tak, to tutaj chyba powinien być `title` z konfiguracji strony.